mindjunkies/courses/forms.py

```
@@ -33,22 +33,23 @@ class Meta:
-        self.fields["course"].widget.attrs.update(
-            {
-                "class": "block w-full p-2 border border-gray-300 rounded-md shadow-sm focus:ring "
-                "focus:ring-blue-500 focus:border-blue-500",
-            }
-        )
+        fields = ['motivation', 'intro_video']
+        widgets = {
+            'motivation': forms.Textarea(attrs={'class': 'form-input mt-1 block w-full p-2 border border-gray-300 rounded-md focus:ring-indigo-500 focus:border-indigo-500'}),
+            'intro_video': forms.ClearableFileInput(attrs={'class': 'form-input mt-1 block w-full p-2 border border-gray-300 rounded-md focus:ring-indigo-500 focus:border-indigo-500'}),
+        }
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)